css/style.css

```diff
@@ -0,0 +1,747 @@
+* {
+  margin: 0;
+}
+
+body {
```

Comment:
- [ ] The text size on your page is smaller than the original design, kindly consider increasing the font size to look like the CC design and be readable on different screens